types/theme-ui/index.d.ts

```diff
@@ -157,14 +154,6 @@ export interface SxProps {
 
 type SxComponent<T extends SxProps = IntrinsicSxElements['div']> = React.ComponentClass<T & { as?: React.ElementType }>;
 
-export const Box: SxComponent;
```

Comment:
Again, the goal is to define component props, not make them exportable from the main package itself.